test/helper.go

```diff
@@ -7,9 +7,14 @@ import (
 	"github.com/doitintl/kube-no-trouble/pkg/judge"
 	"github.com/doitintl/kube-no-trouble/pkg/rules"
 
+	"github.com/rs/zerolog"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+func init() {
```

Comment:
if this is in a helper can we call it in the other packages?

Reply:
Yep, we can, in the init? So what you're suggesting is to:

- refactor this as standard (non-init) function in this package
- call this function in the `init()` of this and other packages?

Basically, it needs to get called in once in every` _test` package. I'm open to reshuffling things around, just want to keep it simple.

Reply:
Moved to another PR